src/index.js

```diff
@@ -1,55 +1,37 @@
 /* eslint-disable no-unused-vars */
-import _ from 'lodash';
+import _, { add } from 'lodash';
 import './style.css';
+import {
+  toDoList, addItem, removeTask, generateElement, update,
+} from './addremove.js';
 /* eslint-enable no-unused-vars */
 
-const toDoList = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Food',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Baith',
-    completed: false,
-    index: 3,
-  },
-  {
-    description: 'Sleep',
-    completed: true,
-    index: 4,
-  },
-];
-
-toDoList.forEach((e) => {
-  const { description, completed, index } = e;
-
-  const ulLists = document.querySelector('.ul-lists');
-
-  const list = document.createElement('li');
-  list.setAttribute('class', 'list');
-  list.setAttribute('id', index);
-  ulLists.appendChild(list);
-
-  const span = document.createElement('span');
-  span.setAttribute('class', 'list-check');
-  list.appendChild(span);
-
-  const checkbox = document.createElement('input');
-  checkbox.setAttribute('type', 'checkbox');
-  checkbox.checked = completed;
-  span.appendChild(checkbox);
-
-  const paragraph = document.createElement('p');
-  paragraph.textContent = description;
-  span.appendChild(paragraph);
-
-  const ellipsis = document.createElement('i');
-  ellipsis.setAttribute('class', 'fa fa-ellipsis-v');
-  list.appendChild(ellipsis);
-});
\ No newline at end of file
+// Add and Remove
+const addBtn = document.querySelector('.add-items-btn');
+const input = document.querySelector('#text-input');
+
+toDoList.forEach((elt) => {
+  generateElement(elt.index, elt.completed, elt.description);
+  const lists = document.querySelectorAll('.ul-lists li');
+  removeTask(lists);
+  update(lists);
+});
+
+const triggerEvent = () => {
+  addItem(input.value, false);
```

Comment:
[Optional] I recommend that you trim the input value before saving it to be sure that a user wouldn't be able to save a string containing only spaces.

![image](https://user-images.githubusercontent.com/20554583/194525353-2bb7e228-48f4-4e4c-a8e8-e545049dadfd.png)